api/app/auth.py

```diff
@@ -28,4 +28,4 @@ async def authenticate(token: str = Depends(oauth2_scheme)):
             status_code=status.HTTP_401_UNAUTHORIZED,
             detail=detail,
             headers={'WWW-Authenticate': 'Bearer'},
-        )
+        ) from exception
```

Comment:
Can you explain the difference between having this and not having?

Reply:
It makes for a better stack trace - as it includes the original exception.

Reply:
It actually came up in linting when I upgraded our packages, I didn't know about it until before today.